### PR TITLE
Commerce 2036 Pricing price list now support Filters and sorts

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
tests should be fixed in upstream now

File: headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/odata/entity/v1_0/PriceListEntityModel.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import com.liferay.portal.odata.entity.EntityModel;
+import com.liferay.portal.odata.entity.StringEntityField;
+
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * @author Alessio Antonio Rendina
```
Comment (ethanbustad) [inline headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/odata/entity/v1_0/PriceListEntityModel.java:28]: ?

**Review by ethanbustad — APPROVED** [review]

File: headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/resource/v1_0/PriceListResourceImpl.java [hunk 2]
```diff
@@ -141,21 +155,21 @@ public PriceList getPriceListByExternalReferenceCode(
...
-				pagination.getStartPosition(), pagination.getEndPosition(),
-				null);
-
-		int totalItems = _commercePriceListService.getCommercePriceListsCount(
-			contextCompany.getCompanyId(), WorkflowConstants.STATUS_APPROVED);
-
-		return Page.of(
-			_toPriceLists(commercePriceLists), pagination, totalItems);
+		return SearchUtil.search(
+			booleanQuery -> booleanQuery.getPreBooleanFilter(), filter,
```
Comment (marco-leo) [inline headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/resource/v1_0/PriceListResourceImpl.java:163]: @andrea-ale-sbarra  is the status a possible filter value?

**Comment by marco-leo** [comment]
sent to https://github.com/liferay/com-liferay-commerce/pull/1662

**Pull request closed without merging**